src/services/FrogbotService.ts

```diff
@@ -0,0 +1,166 @@
+                owner,
+                repo,
+                ref: newBranchRef,
+                sha: latestCommitSha,
+            });
+        } catch (error) {
+            console.error('Error creating branch:', error);
+        }
+    }
+
+    private async addFrogbotWorkflows(owner: string, repo: string, defaultBranch: string) {
+        try {
+            const scanRepoWorkflow = this.scanRepositoryWorkflow(defaultBranch);
+            const pullRequestWorkflow = this.pullRequestWorkflow();
+
+            await Promise.all([
+                this.octokit.rest.repos.createOrUpdateFileContents({
+                    owner,
+                    repo,
+                    path: '.github/workflows/frogbot-scan-repository.yml',
+                    message: `Added frogbot-scan-repository.yml on ${PULL_REQUEST_DATA.branchName}`,
+                    content: Buffer.from(scanRepoWorkflow).toString('base64'),
+                    branch: PULL_REQUEST_DATA.branchName,
+                }),
+                this.octokit.rest.repos.createOrUpdateFileContents({
+                    owner,
+                    repo,
+                    path: '.github/workflows/frogbot-scan-pull-request.yml',
+                    message: `Added frogbot-scan-pull-request.yml on ${PULL_REQUEST_DATA.branchName}`,
+                    content: Buffer.from(pullRequestWorkflow).toString('base64'),
+                    branch: PULL_REQUEST_DATA.branchName,
+                }),
+            ]);
+        } catch (error) {
+            console.error('Error adding workflows:', error);
+        }
+    }
+
+    private async openPullRequest(owner: string, repo: string, defaultBranch: string) {
+        await this.octokit.rest.pulls.create({
+            owner,
+            repo,
+            head: PULL_REQUEST_DATA.branchName,
+            base: defaultBranch,
+            title: PULL_REQUEST_DATA.prTitle,
+            body: PULL_REQUEST_DATA.comment,
+        });
+    }
+
+    private pullRequestWorkflow(): string {
+        return `Name: Frogbot pull request scan"
+jobs:
+  scan-pull-request:
+    runs-on: ubuntu-latest
+    environment: frogbot
+    steps:
+      - uses: jfrog/frogbot@v2.21.13
+        env:
+          JF_URL: \${{ secrets.JF_URL }}
+          JF_ACCESS_TOKEN: \${{ secrets.JF_TOKEN }}
+          JF_GIT_TOKEN: \${{ secrets.GITHUB_TOKEN }}
+`;
+    }
+
+    private scanRepositoryWorkflow(defaultBranch: string): string {
+        return `name: "Frogbot Scan Repository"
+on:
+  push:
+    branches:
+      - "${PULL_REQUEST_DATA.branchName}"
+  workflow_dispatch:
+  repository_dispatch:
+    types:
+      - trigger-frogbot-scan
+  schedule:
+    - cron: "0 0 * * *"
+
+permissions:
+  contents: write
+  pull-requests: write
+  security-events: write
+
+jobs:
+  scan-repository:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        branch: [ "${defaultBranch}" ]
+    steps:
+      - uses: jfrog/frogbot@v2.21.13
```

Comment:
```suggestion
      - uses: jfrog/frogbot@v2
```
Use latest version, instead of the constant